io.github.achetagames.epic_asset_manager.json

```diff
@@ -0,0 +1,99 @@
+{
+  "app-id": "io.github.achetagames.epic_asset_manager",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "40",
+  "sdk": "org.gnome.Sdk",
+  "command": "epic_asset_manager",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.rust-stable"
+  ],
+  "finish-args": [
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--share=network",
+    "--filesystem=host",
+    "--talk-name=org.freedesktop.secrets",
+    "--device=dri",
+    "--env=G_MESSAGES_DEBUG=none"
+  ],
+  "build-options": {
+    "append-path": "/usr/lib/sdk/rust-stable/bin",
+    "build-args": [
+      "--share=network"
+    ],
+    "test-args": [
+      "--socket=x11",
+      "--share=network"
+    ],
+    "env": {
+      "RUST_BACKTRACE": "1"
+    }
+  },
+  "modules": [
+    {
+      "name" : "libadwaita",
+      "buildsystem" : "meson",
+      "config-opts" : [
+        "-Dintrospection=disabled",
+        "-Dgtk_doc=false",
+        "-Dtests=false",
+        "-Dexamples=false",
+        "-Dvapi=false"
+      ],
+      "cleanup" : [
+        "/include",
+        "/lib/pkgconfig"
+      ],
+      "sources" : [
+        {
+          "type" : "git",
+          "url" : "https://gitlab.gnome.org/GNOME/libadwaita.git",
+          "branch" : "main",
```

Comment:
```suggestion
```

A branch move which mean next commit, the build break. Build should be reproducible.